### ISSUE TITLE
Maybe add support for date and time?

### ISSUE THREAD

**Issue by ad-si** [m1l/Everything-Metric#2]
I was just about to build an extension to reformat dates and times to my preferred format, but it somehow feels related enough to this extension, that it should be included here. While ISO8601 isn't part of the SI, it's closesly related. So what do you think?

**Comment by kieronlanning** [comment]
I was about to suggest this too.

I was about to figure out what time the VS 2022 launch date was:

![image](https://user-images.githubusercontent.com/5364423/140740556-d2fc1869-f161-41b9-93f2-573a61d9b386.png)

When I figured there _must_ be a plugin that can convert this info automatically...

**Comment by kazerniel** [comment]
I think the main issue is that with the 10/11/2022 format it's impossible to tell if they meant it the US or international way.

**Comment by m1l** [comment]
I agree, it is hard to tell, especially when people play with separators loosely, using ". / \ -" without following any standard. Even Microsoft announces some of their events as mm.dd.yyyy  which is not in any standard. Things like 12/13/14 could be anything yy/mm/dd, dd/mm/yy, mm/dd/yy or other. I am only comfortable adding something that won't present wrong information, so 3/4 could be March or April, so I would still like users to deploy critical thinking and not rely on a simple algorithm to miss some important date. It could be another project if someone has an idea how to be more sure, but it is not suitable for this one
